src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -456,12 +456,12 @@ export default {
 	the margin applied to the content is added to the total modal width,
 	so here we subtract it to the width and height of the content.
 	*/
-	height: 100%;
```

Comment:
Please note that NcModal has a fixed height (and several pages), which allows content to be scrolled, and depending on the page 'content height' may differ from 'modal height', and that left some place at the bottom:
![image](https://github.com/nextcloud/spreed/assets/93392545/72dfe22a-333c-4122-89cc-f7feffae61a8)

Reply:
Also `height: calc(100% - 98px);` doesn't seem to be needed anymore